### DISCUSSION TITLE
Can not make Barcode/QR manual in form

### DISCUSSION THREAD

**Discussion by muksidin** [Budibase/budibase#9471] [Help] [answered]
Hello, 
I try to create data type Barcode/QR and than i create form manually. Then, i add component Barcode/QR and at field i can not find database with type Barcode/QR. And then i try create data type Text, at field of component Barcode/QR found it. This is bug or not?

Other problems is Barcode can not access camera, i already give it permission and grant but still **Your camera is disabled**.

**Comment by melohagan** [comment, marked as answer]
Hey @muksidin 

I believe this is a bug - thanks for bringing it to our attention. I've raised an issue here: https://github.com/Budibase/budibase/issues/9523

**Reply by muksidin** [reply]
Thanks @melohagan , 
Budibase is the best platform :)
